bindings/wasm/src/account/method_secret.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::crypto::KeyCollection;
+use crate::error::wasm_error;
+use crate::error::Result;
+use identity::account::MethodSecret;
+use identity::core::decode_b58;
+use identity::crypto::PrivateKey;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen(js_name = MethodSecret)]
+pub struct WasmMethodSecret(pub(crate) MethodSecret);
+
+#[wasm_bindgen(js_class = MethodSecret)]
+impl WasmMethodSecret {
+  /// Creates a {@link MethodSecret} object from base58-encoded Ed25519 private key.
+  #[wasm_bindgen]
+  pub fn ed25519_base58(private_key: &str) -> Result<WasmMethodSecret> {
```

Comment:
I made it camalCase. I also prefer enum variants to be capitalized, but it's a method here, it can be confusing for a method name to start with a capital letter.